deploy/registration-service.yaml

```
@@ -222,13 +223,13 @@ objects:
       namespace: ${NAMESPACE}
       labels:
         provider: codeready-toolchain
-        run: registration-service
+        run: proxy-metrics
     spec:
       ports:
         - name: "8082"
```

```suggestion
        - name: metrics
```
we can name the port on the service with `metrics`, too

---

done in [f3e5f2c](https://github.com/codeready-toolchain/registration-service/pull/363/commits/f3e5f2c682f0622f251897ab8541ee32987d6355)